qa/tasks/mgr/dashboard/test_config_options.py

```diff
@@ -0,0 +1,54 @@
+import time
+
+from .helper import DashboardTestCase, authenticate
+
+
+class ConfigOptionsTest(DashboardTestCase):
+
+    @authenticate
+    def test_get_db_config_options(self):
+        data = self._get('/api/config_options')
+        self.assertStatus(200)
+        self.assertEqual(isinstance(data, list), True)
```

Comment:
You can also use 
```python
self.assertSchema(JList(sub_element=...))
```
here, if you want to.

Reply:
What about ``self.assertIsInstance(data, list)`` ?